internal/pkg/service/buffer/worker/distribution/node.go

```diff
@@ -111,13 +111,16 @@ func NewNode(d dependencies, opts ...Option) (*Node, error) {
 	})
 
 	// Create listeners handler
-	n.listeners = newListeners(proc, n.clock, n.logger, n.config)
+	n.listeners = newListeners(n.proc, n.clock, n.logger, n.config)
 
 	// Watch for nodes
 	if err := n.watch(ctx, wg); err != nil {
 		return nil, err
 	}
 
+	// Reset events from the initialization
+	n.listeners.Reset()
+
```

Comment:
Pri inicializacii `distribution.Node`, cakame, kym sa nacitaju vsetky existujuce nodes z etcd.

To je fixnute predchadzajucim PR.

Vramci toho sa nam vygeneruju nejake `node added/removed` events.
V tomto case este neexistuje ziaden listener, kedze sme novu `Node` este nevratili z factory.
Preto tieto eventy zahodime.

Vstupuje do toho `events grouping interval`, ktory by ich poslal napr o `2s` ako zmeny, aj ked to nie su zmeny, je to inicializacia, na ktoru sa caka.